src/Features/Core/Portable/CodeRefactorings/FixAllOccurences/FixAllCodeRefactoringCodeAction.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis.CodeFixesAndRefactorings;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings
+{
+    internal sealed class FixAllCodeRefactoringCodeAction : FixAllCodeAction
+    {
+        internal FixAllCodeRefactoringCodeAction(IFixAllState fixAllState)
+            : base(fixAllState, showPreviewChangesDialog: true)
```

Comment:
who passes 'false' for this btw?

Reply:
It is `false` below: https://github.com/dotnet/roslyn/blob/dbedfa27696dba71d3f5cbe8a956bfa30cfacb21/src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixMultipleCodeAction.cs#L14-L18

I believe this is used by baselining command where we don't want to show a preview/prompt, [doc](https://docs.microsoft.com/visualstudio/code-quality/in-source-suppression-overview?view=vs-2019#suppress-violations-using-a-global-suppression-file):

> From the Analyze menu, select Analyze > Build and Suppress Active Issues on the menu bar to suppress all current violations. Suppressing all current violations is sometimes referred to as "baselining"